tensorflow_addons/losses/contrastive.py

```diff
@@ -36,10 +35,19 @@ def contrastive_loss(
     `a` and `b` with shape `[batch_size, hidden_size]` can be computed
     as follows:
 
-    ```python
-    # y_pred = \sqrt (\sum_i (a[:, i] - b[:, i])^2)
-    y_pred = tf.linalg.norm(a - b, axis=1)
-    ```
+    >>> a = tf.constant([[1, 2],
+    ...                 [3, 4],
+    ...                 [5, 6]], dtype=tf.float16)
+    >>> b = tf.constant([[5, 9],
+    ...                 [3, 6],
+    ...                 [1, 8]], dtype=tf.float16)
+    >>> y_pred = tf.linalg.norm(a - b, axis=1)
+    >>> y_pred
+    <tf.Tensor: shape=(3,), dtype=float16, numpy=
```

Comment:
Can you try to use `# doctest: +NORMALIZE_WHITESPACE`

Reply:
@bhack I just tried adding `# doctest: +NORMALIZE_WHITESPACE` but it seems to make no difference. (Maybe I'm using it wrong?) The interesting thing is that these tests pass for other files (`kappa_loss.py` &  `npairs.py`) where I utilize the same format.

Reply:
`kappa_loss.py` is single line.

Reply:
You find some example in https://pymotw.com/2/doctest/